src/snakeoil/compression/_xz.py

```diff
@@ -0,0 +1,72 @@
+"""
+xz decompression/compression
+
+Where possible, this module defers to cpython's lzma module - if it's not available,
+it defers to executing xz with tempfile arguments to do decompression
+and compression.
+
+Use this module unless it's absolutely critical that lzma module be used.
+"""
+
+__all__ = ("compress_data", "decompress_data")
+
+import multiprocessing
+from functools import partial
+
+from .. import process
+from ..compression import _util
+
+# Unused import
+# pylint: disable=W0611
+
+# if xz can't be found, throw an error.
+xz_path = process.find_binary("xz")
+xz_compress_args = ('-T%i' % multiprocessing.cpu_count(),)
+xz_decompress_args = xz_compress_args
+parallelizable = True
```

Comment:
What happens if I pass this `-T` to an old version of `xz`? Does it ignore it?

Reply:
Yes.